dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -5367,6 +5367,23 @@ paths:
       summary: Search for ToolTester log files
       tags:
       - toolTester
+  /users:
```

Comment:
Hmm, I don't see this new endpoint, maybe forgot to update the openapi.yaml after removing the endpoint?

Reply:
Good catch.